modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-8-release-note.adoc

```diff
@@ -2,6 +2,15 @@
 
 Version 3.1.8 of Sync Gateway delivers the following features and enhancements:
 
+[WARNING]
+--
+
+We have discovered a critical issue in releases 3.1.9 and 3.1.10 impacting skipped sequence behavior. 
+This issue could lead to the caching feed ending up in an infinite loop.
+We suggest all users promptly roll back to release 3.1.8 and await the 3.1.11 patch.
+
+--
+
```

Comment:
Should this admonition go here or in the overall release note page?

Reply:
Hmm, good point. I'd put in in the specific note in the past but perhaps it could do with being more prominent

Reply:
Done